src/components/chip-set/chip-set.scss

```diff
@@ -1,4 +1,5 @@
 @use '../../style/functions';
+@use '../../style/internal/z-index';
```

Comment:
This doesn't seem to be used in the file. We are not using any `z-index`es from this file, here in the styles.

Reply:
right, I will remove it